@navikt/core/react/src/toggle/Toggle.tsx

```diff
@@ -0,0 +1,137 @@
+import cl from "classnames";
+import React, {
+  createContext,
+  forwardRef,
+  HTMLAttributes,
+  useState,
+} from "react";
+import TogglesButton, { ToggleButtonsType } from "./ToggleButton";
+
+export interface ToggleProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, "onChange"> {
+  /**
+   * Toggles.Button elements
+   */
+  children: React.ReactNode;
+  /**
+   * Changes padding and font-size
+   * @default "medium"
+   */
+  size?: "medium" | "small";
+  /**
+   * Acitive value
+   */
+  value?: string[];
+  /**
+   * DefaultActive version if component should handle state
+   */
+  defaultValue?: string[];
+  /**
+   * Returns elements that wants to be active
+   */
+  onChange: (e: string[]) => void;
+  /**
+   * Allows only a single active element
+   */
+  exclusive?: boolean;
+  /**
+   * requires atleast one toggle to be active
+   */
+  required?: boolean;
+  /**
+   * uses all avalaible space
+   */
+  fullWidth?: boolean;
```

Comment:
Kanskje et dumt spørsmål, men: Hva er bruksområdet til fullbredde?

Reply:
Gjør slik at hver knapp prøver å få samme bredde, så får bruker "containe" hva de ønsker maksbredden skal være.

Reply:
Det Sjur ønsker er at alle knappene har samme bredde, noe som ikke er mulig uten at de tar hele bredden. Har satt `fullWidth` til default `true` nå. Har kanskje vært bedre med et mer forklarende navn her, forslag?

Reply:
Om knappene i en gruppe alltid skal ha samme bredde så trengs det vel ikke en prop?

Reply:
Er egentlig denne casen vi tilrettelegger for: 

![Screenshot 2022-01-26 at 09 47 16](https://user-images.githubusercontent.com/26967723/151131646-13c4faf8-0389-4ea2-9122-a9e98a2208e1.png)

Beste har vært en css løsning der alle ble skalert basert på bredeste element, men tror ikke det er en ren css metode for det uten at hele bredden blir brukt.

Reply:
API-et er inspirert av MUI. De har fullWidth default `false`. Vil kanskje være en bedre default for bruk av komponenten.